src/components/Charts/SparklineChart.tsx

```diff
@@ -25,8 +25,8 @@ function SparklineChart({ width, height, tokenData, pricePercentChange, timePeri
   const hasData = pricePoints.length !== 0
   const startingPrice = hasData ? pricePoints[0] : DATA_EMPTY
   const endingPrice = hasData ? pricePoints[pricePoints.length - 1] : DATA_EMPTY
-  const widthScale = scaleLinear().domain([startingPrice.timestamp, endingPrice.timestamp]).range([0, 124])
-  const rdScale = scaleLinear().domain(getPriceBounds(pricePoints)).range([42, 0])
+  const widthScale = scaleLinear().domain([startingPrice.timestamp, endingPrice.timestamp]).range([0, 110])
+  const rdScale = scaleLinear().domain(getPriceBounds(pricePoints)).range([30, 0])
```

Comment:
it would be nice to have these non-zero numbers written as descriptive variables or explained in a comment (if they have meaning)

Reply:
good idea, added some comments for devs.